docs/source/api/references/index.rst

```diff
@@ -5,7 +5,7 @@ API Reference
 
 The released APIs include the necessary components that can build GraphStorm models, training and inference pipelines. For examples about how to use these APIs, please refer to the :ref:`API Programming Examples<programming-examples>`.
 
-.. note:: This API reference does not include the components for multi-task learning, which is still an experimental feature of Graphstorm v0.3, and may need extensive modification. 
+.. note:: This API reference does not include the components for multi-task learning, which is still an experimental feature of Graphstorm v0.3, and may need extensive modification.
```

Comment:
Maybe we want to remove the mention of the version here until MT is out of experimental?

Reply:
We can update it later. Maybe we can remove it from experimental in 0.4